pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -972,6 +972,14 @@ func consistentHashToHashPolicy(consistentHash *networking.LoadBalancerSettings_
 				},
 			},
 		}
+	case *networking.LoadBalancerSettings_ConsistentHashLB_HttpQueryParameterName:
```

Comment:
Can you please add a test case in `TestBuildHttpRoutes` to cover this new setting?

Reply:
Will do

Reply:
done